### PR TITLE
Create auto-merge.yaml

### PR DESCRIPTION
**Pull request by zregvart**
Ref. https://issues.redhat.com/browse/HACBS-2470

### CODE REVIEW

File: .github/workflows/auto-merge.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+---
+name: 'Auto merge'
+'on':
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  auto-merge:
+    uses: enterprise-contract/github-workflows/.github/workflows/auto-merge.yaml@main
```
Comment (lcarva) [inline .github/workflows/auto-merge.yaml:25]: Why not use the `.github` repo since that already has shared workflows?

Comment (lcarva) [reply .github/workflows/auto-merge.yaml:25]: hmm we should probably move the other workflows there then. Having shared workflows in two separate repos is confusing.

Comment (zregvart) [reply .github/workflows/auto-merge.yaml:25]: The [error](https://github.com/zregvart-test/testbed/actions/runs/6378405326) I was getting for reference https://github.com/zregvart-test/.github/blob/main/workflows/auto-merge.yaml

Screenshot 2023-10-02 at 16-31-40 Auto merge · zregvart-test_testbed@b64770c

Comment (lcarva) [reply .github/workflows/auto-merge.yaml:25]: Could it be because it's across orgs/namespaces?

**Comment by zregvart** [comment]
Okay, so this is not merged, why is it showing up [here](https://github.com/enterprise-contract/ec-cli/actions/workflows/auto-merge.yaml)?

**Comment by lcarva** [comment]
> Okay, so this is not merged, why is it showing up [here](https://github.com/enterprise-contract/ec-cli/actions/workflows/auto-merge.yaml)?

Because it's a branch on this repo, instead of a fork.

File: .github/workflows/auto-merge.yaml (hunk 1 above)
Comment (lcarva) [reply .github/workflows/auto-merge.yaml:25]: oy... let's file a story to move the other shared workflows to the new repo then?

**Review by lcarva — APPROVED** [review]

**Pull request merged**